db/seeds.rb

```diff
@@ -20,3 +20,8 @@
 Desk.create name: 'Desk 303', grouping: floor3, occupied: true
 
 Desk.create name: 'Bench in Quad', grouping: carleton, occupied: false
+
+(Grouping.all + Desk.all).each do |elem|
+  elem.coordinate = Coordinate.create x: rand(0..500), y: rand(0..500)
+  elem.save
```

Comment:
I believe you can achieve both of these lines by building the association directly:
```ruby
elem.build_coordinate(x: rand(0..500), y: rand(0..500))
```